### ISSUE TITLE
Glitch appears to hiding an event that I am attempting to log

### ISSUE THREAD

**Issue by Geoge61** [gordonmzhu/questions#103]
### What is the associated Watch and Code lesson? 

Write your response here (required). Version 10 - Click to Delete, Delete Buttons should have access to the todo id at 3:44

### Include a live demo link so that I can run your code. 

Write your response here (required).
https://glitch.com/edit/#!/loving-aries?path=public/client.js:48:32

### Explain the exact steps I need to take to replicate the problem. 

Write your response here (required).
I have followed the coding instructions upto the time log, 3:44.  I gone into the console and repeated th actions shown in the video.  (1) after making the changes to the code, I used "show live" to open the app in another window to test its behavior.  I added an item and checked "elements". The new List Item was in the UL and showed an appropriate id.  (2) I went to console, and clicked the "delete" button.  The event was not logged in the console. (3) I think that this is a problem with glitch because I have noticed that in the right hand corner of the inspect window, next to the console settings button, I get a message for "hidden" everytime I press the delete button.  It tracks the number of times I hit the delete button for any entry.

### What did you expect to happen? 
  
Write your response here (required).I expected the console to log a mouse event as was shown in the video for this lesson.  Nothing is logged when I press delete.

### What actually happened? 

Write your response here (required).
![image](https://user-images.githubusercontent.com/44447515/48090115-af45d480-e1cb-11e8-826b-2880f1ae7fc8.png)

![image](https://user-images.githubusercontent.com/44447515/48090390-5fb3d880-e1cc-11e8-90ad-7875c76fa471.png)

### Describe what you've done so far to troubleshoot the problem. 

Write your response here (required). I have review the lessons for Version 10 three times and tested every step.  I have recoded the lessons for every requirement.  I repeated the testing until I noticed the "hidden" message on the right side of the console window.  

I have recoded and checked the code ad nauseum.  I am certain that the code is correct.  I certain that code is behaving as expected.  I am certain that the problem is with how glitch is responding to the code.

**Comment by Geoge61** [comment]
https://glitch.com/edit/#!/loving-aries?path=public/client.js:1:0

This is the link I have.  It is still there and unchanged.

**Comment by gordonmzhu** [comment]
Your settings are likely different. Make sure they match up with mine.
<img width="1280" alt="screen shot 2018-11-08 at 12 06 52 pm" src="https://user-images.githubusercontent.com/734575/48224529-de388380-e34e-11e8-9637-cc36ded80aa5.png">
<img width="1280" alt="screen shot 2018-11-08 at 12 06 58 pm" src="https://user-images.githubusercontent.com/734575/48224530-de388380-e34e-11e8-89a5-bd2ea37e87d3.png">

**Comment by Geoge61** [comment]
[image: image.png]
[image: image.png]

Hi Gordon,

I appreciate your help.  If you have another idea about what could be
wrong, I would be happy to hear it.  I have learned a lot from your class.
The most important being how to efficiently learn programming.

When I figure out what is causing the problems I am having, I will be
back.  For the moment, I am pressed for time and have to run with a program
in which the problems and related to programming and not the software.
Taking the lessons that I have learned from your organization of Watch &
Code, I have put together a program that I know will work and obtained some
resources that I believe will serve my purposes.

Again, thank you for your time and effort.  I look forward to learning
javascript from you later, but also to gainer further insights into the
programming per se.

God bless,
George Christovich

P.S. A summary of  what I gleaned from your course about learning to
program.
To become a competent programmer I must,
1) learn how to solve problems independently, i.e., be able to teach myself.
2) learn the basics of a programming language and develop the skills
necessary to develop advanced techniques.
3) apply what I have learned in a project of sufficient scope to both
challenge me and display my abilities.

**Comment by gordonmzhu** [comment]
Please re-upload the images. They are not showing up.

**Comment by Geoge61** [comment]
Hey Gordon,

I found the problem. I am running a linux box now, and everything is
working as it should.  I will back in the mines tomorrow.  Thanks for you
help.

I don't know if it is worth warning people about trying to run the course
with a Chromebook or not.  My own experience over the years is that these
incompatibilities are both small and ephemeral.  I stuck with Chromebook
for all these years because the system fit my budget.  I suspect most
people running Chromebook in this sort of environment have the
same problem.  I bought a refurbished Acer C720p and installed Galliumos.
Its optimized for Chromebook hardware, but is a built on Xubuntu.  Total
cost for a linux box, $150, but I needed two, machines to complete the
project.

P.S.  I showed my son, 13yo, your course and he has started PJS.  He likes
your lecture style.
